src/core/public/plugins/plugins_service.ts

```diff
@@ -98,6 +98,8 @@ export class PluginsService implements CoreService<PluginsServiceSetup> {
     return { contracts };
   }
 
+  public async start() {}
```

Comment:
was left empty intentionally to urge to use `start` when needed.

Reply:
addressed in https://github.com/elastic/kibana/pull/35297/commits/c04fdd2e26f483b3e612b854cc1c15a000a9bd13